roles/apim-km/templates/carbon-home/repository/conf/carbon.xml.j2

```diff
@@ -409,11 +409,11 @@
             <!-- Keystore file location-->
             <Location>{{ internal_keystore_location }}</Location>
             <!-- Keystore type (JKS/PKCS12 etc.)-->
-            <Type>{{ internal_keystore_type }}</Type>
+            <Type>JKS</Type>
             <!-- Keystore password-->
-            <Password>{{ internal_keystore_password }}</Password>
+            <Password>wso2carbon</Password>
```

Comment:
Pamatereize this as well.

Reply:
+1 will have to parameterize this for Gateway as well

Reply:
Added https://github.com/wso2/ansible-apim/pull/49/commits/f26ff3b4bf62ab81d6a89ce61855141129c2690f